server/test_devpi_server/test_model.py

```diff
@@ -1028,7 +1028,7 @@ def test_get_indexconfig_lists(xom, key, value, result):
      dict(type="stage")),
 
     ({"volatile": "foo"},
-     dict(type="stage", volatile=True)),
+     InvalidIndexconfig),
 
```

Comment:
Tests with ``"True"`` or ``"true"`` are missing

Reply:
fixed in 5580a5be